src/main/java/Ui/Superhero.java

```diff
@@ -0,0 +1,130 @@
+package Ui;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import WordList.ToLearn;
+import WordList.VocabList;
+import WordList.Vocabulary;
+import WordList.Deadline;
+import WordList.Event;
+
+public class Superhero {
```

Comment:
I like how you personalize the Duke as Superhero haha👍🏻👍🏻 it's quite interesting